setup.py

```diff
@@ -69,7 +69,7 @@
 ext = '.pyx' if USE_CYTHON else '.cpp'
 
 extra_compile_args = {
-    'msvc': ['/std:c++latest', '/EHsc'],
+    'msvc': ['/std:c11', '/EHsc'],
```

Comment:
I believe this is ignored when compiling C++ code (`c11` != `c++11`). Modern MSVC will [default to `/std:c++14`](https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=vs-2019) for C++ code.

Reply:
Got it, so the fix came from removing `c++latest` rather than adding `c11`. Thanks for the sanity check! I will fix